zkevm/Cargo.toml

```diff
@@ -19,6 +19,7 @@ bus-mapping = { git = "https://github.com/scroll-tech/zkevm-circuits.git", branc
 eth-types = { git = "https://github.com/scroll-tech/zkevm-circuits.git", branch = "scroll-dev-0920" }
 zkevm-circuits = { git = "https://github.com/scroll-tech/zkevm-circuits.git", branch = "scroll-dev-0920", features = ["test"] }
 mpt-circuits = { package = "halo2-mpt-circuits", git = "https://github.com/scroll-tech/mpt-circuit.git", branch = "scroll-dev-0902" }
+zktrie = { git = "https://github.com/scroll-tech/zktrie.git" }
```

Comment:
we should never set dep like this. Either use commit or use branch(with compatibility guarantees). .

Reply:
i prefer using branch. So for simple bug fixs, just run cargo upgrade.

Reply:
done, currently we have `dev-1102` branch for cooperate with mpt circuit with `scroll-dev-0902` branch

I consider integrate zktrie into mpt circuit and re-export it to avoid aligning deps in mutiple crates (mpt, hash, and zktrie, etc) later